node/resolvers/search/product.ts

```diff
@@ -197,6 +199,18 @@ export const resolvers = {
       }
       return searchItems
     },
+    priceRange: ({ items: searchItems }: SearchProduct) => {
+      const allSellers = flatten(pluck('sellers', searchItems))
```

Comment:
Can't we replace all of these `flatten`s, `pluck`s and `filter`s to a single reduce operation ?